osv/ecosystems.py

```diff
@@ -144,6 +144,29 @@ def _get_affected_versions(self, versions, introduced, fixed, last_affected,
   def is_semver(self):
     return False
 
+  @property
+  def supports_ordering(self):
+    return True
+
+
+class OrderingUnsupportedEcosystem(Ecosystem):
+  """Placeholder ecosystem helper for unimplemented ecosystems"""
+
+  def sort_key(self, version):
+    raise ValueError("Ecosystem helper does not support sorting")
```

Comment:
Would `NotImplementedError` seems like it would make more sense in this class?

Reply:
+1